CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 4.1.1 (2020-07-07)
+- Fixed issue where fuzzy date could return incorrect data when sending in a day value of the 31st. [#181](https://github.com/blackbaud/skyux-datetime/pull/181) (Thanks [@Blackbaud-CoreyArcher](https://github.com/Blackbaud-CoreyArcher)!)
```

Comment:
Would this work?

`Fixed an issue where the fuzzy date pipe returns incorrect data when the day value is 31.`

Reply:
I like the latter.

Reply:
Normally I would go with the second one as it is more concise. However, in this case the 31 date didn't seem to _always_ break and so lets go with the first :) I'll update.

Reply:
@blackbaud-johnly updated :)

Reply:
I didn't see Alex's message until after my comments were added. I'm fine if @blackbaud-johnly feels like we should use the first. Feel free to update it to that if you like that better.

Reply:
I liked the second one better, but either one worked for me, so I just approved the PR. Thanks!